root/static/scripts/common/components/EntityLink.js

```diff
@@ -448,6 +459,15 @@ component EntityLink(
     );
   }
 
+  if (creditedAs !== null) {
```

Comment:
You could use `nonEmpty(creditedAs)` here, that's really what we want anyway I think :)